picard/ui/infodialog.py

```diff
@@ -150,11 +163,28 @@ class ArtworkTableExisting(ArtworkTable):
     artwork_columns = ('orig', 'new',)
 
 
+class ArtworkTableExistingExternal(ArtworkTable):
```

Comment:
Do we really need separate classes for each case? As zas noted above it should also consider "Embed cover images into tags" and hide that column. Implementing this with yet another class will increase the combinations and the number of classes needed.

Maybe the better approach is then to hide no needed columns with https://doc.qt.io/qt-6/qtableview.html#setColumnHidden